content/projects/gsoc/2019/application.md

```diff
@@ -0,0 +1,120 @@
+---
+:layout: simplepage
+:title: "Jenkins GSoC application draft, 2019"
+:tags:
+- gsoc
+- gsoc2019
+---
+
+<!-- This file uses Markdown intentionally.
+     Google's GSoC site uses Markdown as an engine, using Markdown here allows copy-pasting
+ -->
+
+The data below represents the 2019 GSoC application draft.
+Please feel free to contribute to it by proposing pull requests against the page.
+
+### Organization Profile
+
+* Tagline (up to 80 symbols): Jenkins is an open-source automation server
+* Technology tags: java, groovy, javascript
+* Topic Tags: continuous integration, continuous delivery, developer tools, devops
+* Ideas list: https://jenkins.io/projects/gsoc/2019/project-ideas/ 
+* Short description:
+Jenkins is the open source automation server. Built primarily in Java, it provides hundreds of plugins to support building, testing, deploying and automating virtually any project.
+
+#### Long description (Markdown)
+
+[Jenkins](https://jenkins.io/), originally founded in 2006 as "Hudson", is one of the leading automation servers available. Using an extensible, plugin-based architecture developers have created hundreds of plugins to adapt Jenkins to a multitude of build, test, and deployment automation workloads. Jenkins core is open-source ([MIT License](http://www.opensource.org/licenses/mit-license.php))
+
+The project has about 400 active contributors working on Jenkins core, plugins, website, project infrastructure, localization activities, etc. In total we have around 2000 different components including plugins, libraries, and various utilities. The main languages in the project are Java, Groovy and JavaScript, but we also have components written in other languages.
+
+This year we invite students to join the Jenkins community and to work together on Jenkins plugins in order to improve Jenkins user experience and reliability.
+
+
+#### Proposal process:
+
+In order to apply to the organization, please follow the [guidelines posted on our website](https://jenkins.io/projects/gsoc/students/#student-application-process). 
+
+Before submitting please go through the page (including the expectations section) and through the [Student guide](https://google.github.io/gsocguides/student/). Participating in Google Summer of Code is a serious commitment requiring an almost full-time dedication over several months. If it may overlap with your study and other commitments, we recommend to think twice before applying.
+
+If you have any questions about the application process, please feel free to contact us via the “jenkinsci-gsoc-orgs@googlegroups.com” mailing list.
```

Comment:
```suggestion
If you have any questions about the application process, please feel free to contact us via the “jenkins-gsoc-2019-org-admins@googlegroups.com” mailing list.
```

Reply:
was also suggested as a change in the diff, should be fixed now